contextualhelp/data.json

```diff
@@ -662,7 +712,7 @@
         ]
       },
       {
-        "route_name": "MESSAGE_DETAIL",
+        "route_name": "MESSAGE_DETAIL_PAGINATED",
```

Comment:
@CrisTofani is this the right route. Locally now FAQ seems be rendered with correct data.